src/main/java/edu/wisc/my/ltiproxy/dao/LTILaunchPropertyFileDao.java

```diff
@@ -5,12 +5,13 @@
 
 import com.fasterxml.jackson.core.JsonParseException;
 import com.fasterxml.jackson.databind.JsonMappingException;
+import com.google.common.collect.Multimap;
 
 public interface LTILaunchPropertyFileDao{
     
     public Map<String, String> getLaunchParameters(String key) throws JsonParseException, JsonMappingException, IOException;
     
-    public Map<String, String[]> getHeadersToReplace(String key) throws JsonParseException, JsonMappingException, IOException;
+    public Multimap<String, String> getHeadersToReplace(String key) throws JsonParseException, JsonMappingException, IOException;
```

Comment:
Let's go to 2.  I can see the API being these three things: Java Signatures, Rest Endpoints, and Runtime Configuration.  While a normal adopter would typically only care about the endpoints and configuration, I think it does feel a little wrong to say that the signatures don't matter.

I didn't intend to major version bump it, but I do think Multimap (or at least `Map<String, Collection<String>>`) is the correct type for this, so I don't necessarily want to drop it.

Versions are cheap, let's go 2.